packages/table/src/regions.ts

```diff
@@ -258,6 +258,24 @@ export class Regions {
     }
 
     /**
+     * Returns true if the regions contain a region that has FULL_TABLE cardinality
+     */
+    public static hasFullTable(regions: IRegion[]) {
+        if (regions == null) {
+            return false;
+        }
+
+        for (const region of regions) {
+            const cardinality = Regions.getRegionCardinality(region);
+            if (cardinality === RegionCardinality.FULL_TABLE) {
+                return true;
```

Comment:
seems like a good opportunity for `regions.some()`

oh i see now this is copy-pasta. would be a fun refactor though :man_shrugging: